File: Converge/ClientApp/src/types/VenueDetails.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

import BaseVenue from "./BaseVenue";
import YelpLocation from "./YelpLocation";
import OperationHoursForWeek from "./OperationHoursForWeek";

interface VenueDetails extends BaseVenue {
  alias?: string;
  isClaimed?: boolean;
  isClosed?: boolean;
  location?: YelpLocation;
  photos?: string[];
  operatingHours?: OperationHoursForWeek;
}

export default VenueDetails;

File: Converge/Models/VenueDetails.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using System.Collections.Generic;

namespace Converge.Models
{
    public class VenueDetails : BaseVenue 
    {
        public string Alias { get; set; }

        public bool IsClaimed { get; set; }

        public bool IsClosed { get; set; }

        public YelpLocation Location { get; set; }

        public List<string> Photos { get; set; }

        public OperationHoursForWeek OperatingHours { get; set; }
    }
}
